content/blog/2023/unintuitive-objects.md

```diff
@@ -0,0 +1,475 @@
+is to provide people (and dolls)
+a place to sit down.
+The purpose of the digital article object
+on the blog is
+to convey information to readers
+about specific topics of interest.
+
+> "All humans think in objects -- designers, end-users, and those who
+> painted the caves of Chauvet. So, as we make our user experiences
+> more object-oriented, we also make them more *intuitive*."
+>
+>--*Sophia Prater*, [ooux.com](https://www.ooux.com/what-is-ooux)
+
+With that definition of objects in mind,
+let's look at what makes a digital object *unintuitive*
+for people using the web.
+Though it may seem backward,
+looking at examples of digital objects gone wrong
+is a really impactful way
+to learn how to make them more intuitive.
+
+## Unintuitive Objects on the Web
+
+There are four type of unintuitive objects:
+**Broken**, **Isolated**, **Masked**, and **Shapeshifter**.
+To find examples of these unintuitive objects,
+I decided to go spelunking through the website
+of one of my favorite places to visit --
+the Denver Zoo.
+The photography and graphic design are lovely,
+and what a great excuse
+to pore over pictures of cute animals!
+*ANIMAL*, *HABITAT*, and *DEMO*
+are three important digital objects
+on the Denver Zoo's website.
+
+### Broken Objects
+*Objects that are not directly manipulatable.*
+
+A *broken object* is an object
+that isn’t directly manipulatable
+within a digital space.
+Let's look at an example
+on the Denver Zoo's site.
+
+One of my favorite things to do at the zoo
+is attend demonstrations like the feeding of the penguins,
+so DEMO is a really useful object
+for the zoo to include on its website.
+Hooray!
+Unfortunately, DEMO has been incorporated
+into this website in a broken way.
+
+For example,
+here are instances of the DEMO object
+in a list.
+
+{{ embed.img(
+  src='blog/2023/broken-event.jpg',
+  alt='penguin demo at two times in a list of events'
+) }}
+
+As someone who would like to attend the penguin DEMO,
+I want to click on an individual DEMO
+to see more details,
+like a description or a map
+showing the way to the location,
+but I can’t.
+There are also two different times
+when the penguin DEMO happens,
+and a variety of locations
+where DEMOS happen.
+As a person planning my visit to the zoo,
+it would be useful to see all the time options
+for one DEMO grouped together,
+or see only the DEMOS happening at a specific time,
+or all the DEMOS happening at one location.
+But I can’t sort or filter the DEMOS
+by time or location --
+I can’t take the actions I want to take
+when I encounter this DEMO object in this system.
+As a regular visitor
+to the zoo site,
+I would describe this experience
+as *unintuitive*.
+
+If I were redesigning this DEMO list
+using the object-oriented method,
+I would identify the DEMO object
+at the very beginning of the project
+and list all the actions people would like to take
+before designing a landing page like this
+or making any mockups at all.
+That way,
```

Comment:
@mirisuzanne I thought about that, but decided not to get into it because I think that would require explaining object maps. If I write a second article about object maps, this would be a good place to mention it and link to that.

Reply:
Yeah, I thought there might be a middle ground. Show a fragment of documentation without getting into all the context and process around it? But either way.